thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/AlertFilterType.java

```diff
@@ -1,5 +1,5 @@
 package com.linkedin.thirdeye.detector.email.filter;
 
 public enum AlertFilterType {
-  DUMMY, ALPHA_BETA
+  DUMMY, ALPHA_BETA, ALPHA_BETA_LOGISTIC
```

Comment:
Second this, we do not put anything that is in 'anomaly-detection lib' in thirdeye-pinot. This is a good isolation strategy (for internal and open source use cases).